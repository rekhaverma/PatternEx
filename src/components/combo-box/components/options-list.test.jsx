import React from 'react';
import OptionList from './options-list.component';

/**
 * @status: Complete
 * @sign-off-by: Alex Andries
 */
describe('<OptionList />', () => {
  it('should match with snapshot', () => {
    const props = {
      options: ['Test', 'Another Test'],
      onClick: jest.fn(),
    };
    const wrapper = shallow(<OptionList {...props} />);

    expect(wrapper).toMatchSnapshot();
  });
});
