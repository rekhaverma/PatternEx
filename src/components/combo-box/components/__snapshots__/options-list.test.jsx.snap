// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OptionList /> should match with snapshot 1`] = `
<div
  className="comboBox__optionList"
>
  <Scrollbars
    autoHeight={true}
    autoHeightMax={200}
    autoHeightMin={0}
    autoHide={false}
    autoHideDuration={200}
    autoHideTimeout={1000}
    hideTracksWhenNotNeeded={false}
    renderThumbHorizontal={[Function]}
    renderThumbVertical={[Function]}
    renderTrackHorizontal={[Function]}
    renderTrackVertical={[Function]}
    renderView={[Function]}
    tagName="div"
    thumbMinSize={30}
    universal={false}
  >
    <Option
      className="selectBox__option"
      isActive={false}
      key="0"
      onClick={[Function]}
      option="Test"
    />
    <Option
      className="selectBox__option"
      isActive={false}
      key="1"
      onClick={[Function]}
      option="Another Test"
    />
  </Scrollbars>
</div>
`;
